content/docs/setup/kubernetes/additional-setup/config-profile/index.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Should the directory be config-profiles?

Reply:
Done